images/assets/pulp-api

```diff
@@ -1,44 +1,7 @@
-#!/bin/bash -x
-
-mkdir -p /var/lib/pulp/media \
-         /var/lib/pulp/assets \
-         /var/lib/pulp/tmp
```

Comment:
/var/lib/pulp is mounted as a volume. This must happen at runtime. I wonder if the services using it are creating these dirs on demand? How does it even work for pulp these days @decko ?

Reply:
> /var/lib/pulp is mounted as a volume. This must happen at runtime. I wonder if the services using it are creating these dirs on demand? How does it even work for pulp these days @decko ?

Ok, so it was being re-created after the /var/lib/pulp being mounted.
Should it have it's mount point?

Reply:
> Our compose file mounts a volume on /var/lib/pulp. So i suspect one of the setup processes is probably recreating these dirs?

Yes, we mount, but we're not recreating those directories during the runtime. Since this is a tmp dir, would make send for pulp itself create it if it's not found?